roles/docker/tasks/main.yml

```diff
@@ -15,14 +15,14 @@
     sed -i 's/--storage-opt dm.use_deferred_deletion=true//' /etc/sysconfig/docker-storage
   when: docker_storage_check.stat.exists | bool and not docker_version_result | skipped and docker_version_result.stdout | default('0.0', True) | version_compare('1.9', '>=') and docker_version | version_compare('1.9', '<')
 
-- name: Downgrade docker if necessary
-  command: "{{ ansible_pkg_mgr }} downgrade -y docker-{{ docker_version }}"
+- name: Remove docker to downgrade
+  command: "{{ ansible_pkg_mgr }} remove -y docker-*"
```

Comment:
Yeah, docker-1.9.1-40 added a subpackage docker-common to enable the docker-latest stuff. A transaction seems to work. I'll update the PR.
```
yum shell
ts
remove docker-*
install docker-*1.8.2
run
```